### PR TITLE
Access commands

### PR DESCRIPTION
**Pull request by naorgu**

### CODE REVIEW

File: pkg/access/commands/list.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+				return err
+			}
+
+			accessSessions, _, err := client.ClientAPI.AccessSessionsAPI.ListAccessSessions(cmd.Context()).Execute()
+			if err != nil {
+				return err
+			}
+
+			table := uitable.New()
+			table.AddRow("ID", "NAME", "INTEGRATION NAME", "INTEGRATION TYPE", "TYPE")
```
Comment (DmitriyO) [inline pkg/access/commands/list.go:29]: please show to Ofir to make sure it's what we want

File: pkg/access/commands/list.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+			for _, session := range accessSessions.Data {
+				table.AddRow(session.Id, session.Name, session.Integration.Type, session.Type.Name)
```
Comment (DmitriyO) [inline pkg/access/commands/list.go:31]: the integration name is missing?

File: pkg/access/commands/use.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+				output = accessDetails.GetCli()
+			case "link":
+				link := accessDetails.GetLink()
+				output = link.GetUrl()
+			case "instructions":
+				output = accessDetails.Instructions
+			case "json":
+				outputBytes, _ := json.Marshal(accessDetails)
+				output = string(outputBytes)
+			}
```
Comment (DmitriyO) [inline pkg/access/commands/use.go:46]: ofir raised some interesting point for the cli
if the default output is cli, and it's a command,
maybe by default we want to run it, and have a flag for "--no-auto-run"
or maybe have default to show but a flag for "--auto-run"

File: pkg/access/commands/access.go [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func Access() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "access",
+		GroupID: Group.ID,
+		Short:   "List all access sessions",
```
[the remark this answers is not in the document]
Comment (naorgu) [reply pkg/access/commands/access.go]:
```kotlin
func Access() *cobra.Command {
	cmd := &cobra.Command{
		Use:     "access",
		GroupID: Group.ID,
		Short:   "Access sessions root command",
		RunE: func(cmd *cobra.Command, args []string) error {
			fmt.Println("you must specify a subcommand: use, list, reset")
			return nil
		},
	}

	return cmd
}
```

File: pkg/access/commands/use.go (hunk 3 above)
Comment (naorgu) [reply pkg/access/commands/use.go:46]: not sure i understand

**Review by DmitriyO — APPROVED** [review]

**Pull request merged**